client/js/_admin/components/lti_installs/course_install_row.jsx

```diff
@@ -0,0 +1,60 @@
+import React                     from 'react';
+import {
+  createExternalToolCourses,
+  deleteExternalToolCourses,
+} from '../../../libs/canvas/constants/external_tools';
+
+export default function CourseInstallRow(props) {
```

Comment:
same here abstract out functions and put them in parent class and pass as prop, or make this a class if the parent is becoming unwieldy